theboss/simulation_strategies/simulation_strategy_factory.py

```diff
@@ -138,12 +138,35 @@ def _generate_uniform_losses_strategy(self) -> UniformLossSimulationStrategy:
             self._experiment_configuration.uniform_transmissivity
         )
 
-    def _generate_r_cliffords_strategy(self) -> CliffordsRSimulationStrategy:
```

Comment:
Why did you remove the return type? It's still the same (but the method my raise some errors). I think it should stay.

Reply:
I couldn't have it here, since the class `CliffordsRSimulationStrategy` is not imported at this stage yet.